database/postgres/postgres.js

```diff
@@ -0,0 +1,39 @@
+const {Client} = require('pg');
+const client = new Client({
+    host: "localhost",
+    user: "root",
+    database: 'updatessdc'
+});
+
+client.connect(err => {
+    if (err) {
+        console.error('connection error-', err);
+    } else {
+        console.log('connected');
+    }
+});
+
+
+const createDataProject = function(start, end, file) {
+    for (let i = 0; i < 5000; i++) {
+    var insertArr = [];
```

Comment:
const?

Reply:
I'm not using this for data generation, commenting out code